app/controllers/component-tree.js

```diff
@@ -194,10 +194,18 @@ class RenderItem {
     this.renderNode = renderNode;
   }
 
+  get args() {
+    return this.renderNode.args;
+  }
+
   get id() {
     return this.renderNode.id;
   }
 
+  get isCurlyInvocation() {
+    return this.renderNode && this.renderNode.args && this.renderNode.args.positional;
+  }
+
```

Comment:
Minor but can you move both `get args` and `get isCurlyInvocation` to _after_ `get name`? (Trying to maintain roughly [this ordering](https://github.com/emberjs/ember-inspector/blob/master/ember_debug/libs/render-tree.js#L28-L38) in all places for consistency and make things slightly easier to find).

Also, don't need to check `this.renderNode` as it's guaranteed to be present.